systems/WikibaseClient/03-Context_and_Scope.md

```diff
@@ -12,3 +12,12 @@
 ### Technical Context
 
 ![WikibaseClient technical context diagram](./diagrams/03-technical-context.drawio.svg)
+
+| Component                                                                     | Description                                                                                                                                                              |
+| ----------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| Lua/Parser function                                                           | [Lua bindings](../overview/12-Glossary.md#lua) or parser functions invoked through [Wikitext](../overview/12-Glossary.md#wikitext) to embed Entity data in Wiki articles |
+| [ChangeHandler](./05-Building_Block_View.md#entity-change-notifications)      | Receives notifications to update pages when an Item is changed on the Repo                                                                                               |
+| [UpdateRepoHookHandler](./05-Building_Block_View.md#linked-site-page-changes) | Notifies the Repo to update the connected Item when a page is moved or deleted                                                                                           |
+| [Data Bridge](./05-Building_Block_View.md#client-side-item-edits)             | A frontend component enabling Repo edits on the Client via the Repo API                                                                                                  |
+| [Special Pages](./05-Building_Block_View.md#user-interfaces)                  | User interfaces for viewing entity meta data                                                                                                                             |
+| [APIs](./05-Building_Block_View.md#apis)                                      | Wikibase's extensions to MediaWiki's action API exposing Entity (meta) data                                                                                              |
```

Comment:
my ambiguity alarm triggers here, on "extensions" - do we have a better word with the same meaning?

Reply:
Changed the wording!